test/myFTClient.spec.js

```diff
@@ -864,4 +872,159 @@ describe('myFT Client proxy', () => {
         .catch(done);
     });
   });
+
+    //new orgainic v3/kat methods
+    describe('Followed_by_Kat concepts', () => {
+
+      afterEach(done => {
+          nock.cleanAll();
+          done();
+      });
+
+
+      const relProps = Object.assign({}, myFT.followedProperties, {byTool: 'myFTClient.spec', isTest: true});
+      const userConcepts = require('./mocks/fixtures/userFollowedConcept');
+      const katConcepts = require('./mocks/fixtures/userFollowsConceptByKat');
+      const followedByKatRes = require('./mocks/fixtures/followByKatResp');
+    //  const groupConcepts = require('./mocks/fixtures/groupFollowedConcept');
+
+      it('Should set concept(s) follows by a user ', done => {
+          nock(baseUrl)
+            .post(`/kat/user/follows`)
+            .query({noEvent: 'true', waitForPurge: 'false'})
```

Comment:
Do the query items have any bearing on what this endpoint would normally return? If not, you could just pass `true` to `query()`.